tests/test_asciidoc.py

```diff
@@ -0,0 +1,604 @@
+# This file is part of python-markups test suite
+# License: 3-clause BSD, see LICENSE file
+# Copyright: (C) Dmitry Shachnev, 2012-2021
+
+import unittest
+from markups.asciidoc import AsciidocMarkup
+
+
+@unittest.skipUnless(
+    AsciidocMarkup.available(),
+    'asciidoc3.py and/or lxml not available')
+class AsciidocTextTest(unittest.TestCase):
+    def test_basic(self):
+        self.maxDiff = None
+        markup = AsciidocMarkup()
+        converted = markup.convert(Basic_text)
+        text = converted.get_document_body()
+        title = converted.get_document_title()
+        stylesheet = converted.get_stylesheet()
+        stylesheet = stylesheet.replace('\r\n', '\n')
+        html_expected = Basic_html
+        title_expected = 'Hello, world!'
+        stylesheet_expected = Basic_stylesheet
+        # debug
+        # with open('junk.html', 'w') as outfile:
+        #     outfile.write(text)
+        # import pdb; pdb.set_trace()
+        # end debug
+        # using lxml requires replacing EOL chars
+        text = text.replace('\r\n', '\n')
+        self.assertEqual(html_expected[:500], text[:500])
+        self.assertEqual(title_expected, title)
+        self.assertEqual(stylesheet_expected[:300], stylesheet[:300])
+
+
+#
+# =================================================================
+#
+# Data to be used for comparison of correct results.
+#
+
+Basic_text = '''\
+= Hello, world!
+:toc:
+
+== Some subtitle
+
+This is an example *asciidoc* document.
+'''
+
+Basic_html = '''\
+<div id="header">
+<h1>Hello, world!</h1>
+<div id="toc">
+  <div id="toctitle">Table of Contents</div>
+  <noscript><p><b>JavaScript must be enabled in your browser to display the table of contents.</b></p></noscript>
+</div>
+</div>
+<div id="content">
+<div class="sect1">
+<h2 id="_some_subtitle">Some subtitle</h2>
+<div class="sectionbody">
+<div class="paragraph"><p>This is an example <strong>asciidoc</strong> document.</p></div>
+</div>
+</div>
+</div>
+<div id="footnotes"><hr></div>
+<div id="footer">
+<div id="footer-text">
+Last updated 2021-12-29 13:05:50 PST
```

Comment:
This means the test depends on file modification date, and on the user's timezone (which is even worse).

Can you either exclude this text from generated documents, or make the test not rely on its value?